### PR TITLE
Fb database test

### PR DESCRIPTION
**Pull request by Ian-Stewart-Binks**
This pull request demonstrates how to insert user information into a database based on a Facebook authenticated Facebook ID.

This code is built on top of the previous pull request, #274.

### CODE REVIEW

File: hs/server.hs [hunk 1]
```diff
@@ -10,12 +10,15 @@ import Happstack.Server
 import GridResponse
 import GraphResponse
 import AboutResponse
+import qualified Data.Text.Lazy as L
```
Comment (david-yz-liu) [inline hs/server.hs]: What from Data.Text.Lazy are you using?

File: hs/server.hs [hunk 2]
```diff
@@ -164,9 +167,18 @@ createJSONResponse jsonStr = toResponseBS (BS.pack "application/json") jsonStr
 args :: String -> FB.Argument
 args code = ("code", BS.pack code)
 
--- | Retrieves the user's email.
+-- | Retrieves the user's email.s
```
Comment (david-yz-liu) [inline hs/server.hs]: Extra letter.

File: hs/server.hs (hunk 1 above)
Comment (Ian-Stewart-Binks) [inline hs/server.hs]: Nothing. Changed.

File: hs/server.hs (hunk 2 above)
Comment (Ian-Stewart-Binks) [inline hs/server.hs]: Fixed.

**Pull request merged**